index.html

```diff
@@ -35,36 +35,36 @@ <h1 class="headline-lg m-3 logo">
         </nav>
       </div>
     </header>
-    <section class="promotion-container mt-8 d-flex align-center">
-      <div class="p-8">
+    <section class="promotion-container d-flex align-center justify-center">
+      <div class="p-8 promo-data-wrapper">
         <h1 class="headline-lg promo-text-heading">Build Your UI Faster</h1>
-        <p class="text-md w-75 mt-5">
+        <p class="mt-5 promo-text">
           Build beautiful web interfaces by copy and pasting the components at
           one click. Cool UI is user-friendly,customizable and it has powerful
           utility classes.
         </p>
         <div class="d-flex mt-3">
           <button
-            class="btn btn-primary font-wt-bold rounded-sm p-5 m-2"
+            class="btn btn-primary rounded-sm text-sm font-wt-bold p-5"
           >
             <a href="./pages/docs.html" class="no-style-link text-sm">
               Documentation
             </a>
           </button>
           <button
-            class="btn btn-secondary font-wt-bold rounded-sm p-5 m-2"
+            class="btn btn-outline btn-primary-outline font-wt-bold rounded-sm text-sm p-3 ml-6"
           >
-            <a href="#" class="no-style-link text-sm"> Installation </a>
+            <a href="https://github.com/viru2001/Cool-UI" target="_blank" class="no-style-link text-sm"> Github </a>
           </button>
         </div>
       </div>
 
-      <div class="promotional-image-wrapper">
-        <img class="w-75" src="./assets/landing-page-illustartion.svg" alt="" />
+      <div class="promotional-image-wrapper m-6">
+        <img id="promo-img" class="img-responsive" src="./assets/landing-page-illustartion.svg" alt="" />
```

Comment:
can give descriptive value to alt attribute like "Promo image"

Reply:
ok will add it